app/cafe.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor['name']} is not vaccinated.")
```

Comment:
The `visit_cafe` method should return a string in all cases, including when exceptions are raised. Currently, it raises an exception without returning a string. Consider modifying the exception handling to ensure a string is returned as required by the task description.